examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/AppParameters.java

```diff
@@ -19,8 +19,14 @@
 
 public class AppParameters {
   public static final int MIN_ROTATING_DEVICE_ID_UNIQUE_ID_LENGTH = 16;
+  private static final int TEST_SETUP_PASSCODE = 20202021;
+  private static final int TEST_DISCRIMINATOR = 0xF00;
+  private DACProvider TEST_DAC_PROVIDER = new DACProviderStub();
 
   private byte[] rotatingDeviceIdUniqueId;
+  private DACProvider dacProvider = TEST_DAC_PROVIDER;
+  private int setupPasscode = TEST_SETUP_PASSCODE;
+  public int discriminator = TEST_DISCRIMINATOR;
```

Comment:
you have a getter and a setter, so this does not need to be public

Reply:
Thanks for catching this. The first two highlighted attributes were already private, and I have now made the discriminator private as well.